metriq/client.py

```diff
@@ -442,3 +443,17 @@ def method_names_get(self) -> List[Method]:
             Method(**r)
             for r in response["data"]
         ]
+
+    @handler
+    def platform_add(self, environment: EnvironmentCreateRequest) -> Environment:
+        """Add an environment.
+
+        Args:
+            environment (EnvironmentCreateRequest): Environment create request.
+
+        Returns:
+            Environment: Created environment.
+        """
+        response = self.http.post("/task/", data=environment)
+        print(response["message"])
```

Comment:
`print()` is for debugging and isn't necessary when you're done testing. Please remove it.

Reply:
Done